doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -642,6 +642,31 @@ The `mz_sessions` table contains a row for each active session in the system.
 | `client_ip`      | [`text`]                       | The IP address of the client that initiated the session.                                                                  |
 | `connected_at`   | [`timestamp with time zone`]   | The time at which the session connected to the system.                                                                    |
 
+
+## `mz_network_policies`
+The `mz_network_policies` table contains a row for each network pocily.
+
+<!-- RELATION_SPEC mz_internal.mz_network_policies -->
+| Field            | Type                  | Meaning                                                                                                         |
+| -----------------| ----------------------| --------                                                                                                        |
+| `id`             | [`text`]              | The ID of the network policy.                                                                                   |
+| `name`           | [`text`]              | The name of the network policy.                                                                                 |
+| `owner_id`       | [`text`]              | The role ID of the owner of the network policy. Corresponds to [`mz_catalog.mz_roles`](../mz_catalog#mz_roles). |
+| `privileges`     | [`mz_aclitem array`]  | The privileges belonging to the network policy.                                                                 |
+| `oid`            | [`oid`]               | A [PostgreSQL-compatible OID][`oid`] for the network policy.                                                    |
+
+## `mz_network_policy_rules`
+The `mz_network_policy_rules` table contains a row for each network pocily.
```

Comment:
This description is identical to `mz_network_policies`, is there a better way to differentiate them?

Reply:
how about:
`The `mz_network_policy_rules` table contains a row for each network policy rule.`

Reply:
I think that works.